### PR TITLE
Disable flow control

### PR DESCRIPTION
**Pull request by troglobit**
## Description

 - Disable flow control by default on Ethernet ports, both with and without autoneg
 - Disable lldpd on `dsa0` interface, added to `board/common` in case of future devices

## Checklist

Tick relevant boxes, this PR is-a or has-a:

- [X] Bugfix
  - [ ] Regression tests
  - [x] ChangeLog updates (for next release)
- [ ] Feature
  - [ ] YANG model change => revision updated?
  - [ ] Regression tests added?
  - [ ] ChangeLog updates (for next release)
  - [ ] Documentation added?
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (please detail in commit messages)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: src/confd/src/ietf-interfaces.c [hunk 1]
```diff
@@ -676,7 +698,7 @@ static int netdag_gen_ethtool_autoneg(struct dagger *net, struct lyd_node *cif)
 	if (!fp)
 		return -EIO;
 
-	fprintf(fp, "ethtool --change %s autoneg ", ifname);
+	fprintf(fp, "ethtool --change %s  autoneg ", ifname);
```
Comment (wkz) [inline src/confd/src/ietf-interfaces.c:701]:
```suggestion
	fprintf(fp, "ethtool --change %s autoneg ", ifname);
```
Nit

**Review by wkz — APPROVED** [review]
Looks great! Have you tested it on any box with `mv88e6xxx` ports?

**Comment by troglobit** [comment]
> Looks great! Have you tested it on any box with `mv88e6xxx` ports?

Thanks!  Unfortunately no.  Maybe you can help me best started with the ones we've mapped up in the lab tomorrow?

**Comment by wkz** [comment]
> > Looks great! Have you tested it on any box with `mv88e6xxx` ports?
> 
> Thanks! Unfortunately no. Maybe you can help me best started with the ones we've mapped up in the lab tomorrow?

You bet! :+1:

**Pull request merged**